certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -2,45 +2,43 @@
 **  PowerOn Name:       BANNO.CDRENEW.V1.POW
 **  Letterfile Name:    BANNO.CDRENEW.V1.CFG
 **
-**  Copyright 2020 Jack Henry and Associates
+**  Copyright 2020-2022 Jack Henry and Associates
 **
 **  This Banno service PowerOn allows the user to update their
 **  Certificate maturity action.
 **
-**  Persistant Variable Use:
-**    @RGUSERCHR1 All States: Target account and share ID**
-**    @RGUSERCHR2 PROCESSDATA State: Transfer account and share/Loan ID
-**    @RGUSERCHR3 PROCESSDATA State: Transfer Amount
-**    @RGUSERCHR4 Store account number from @RGUSERCHR1 PRELOADDATA State
-**    @RGUSERCHR5 Store share ID from @RGUSERCHR1 PRELOADDATA State
+**  Program states and JSON returns
+**  --------------------------------------------------------------------
+**    STATE: GETPRELOADDATA
+**     PASS:  @RGUSERCHR1 Target account and share ID
+**  PROCESS:  Take account & share ID provided by the UX and based
+**            upon the parameter settings from the configuration Letter
+**            file, return current share state and valid options.
+**   RETURN:  Provide UX with the following data.
+**            Current cert maturity settings.
+**            List of CU approved options for the member to select from.
+**            CU custom messages, TOCs to display to the member.
+**            List of shares available for transfer into or out.
 **
-**    @RGUSERNUM1 PROCESSDATA State: Member option selected
-**    @RGUSERNUM2
-**    @RGUSERNUM3
-**    @RGUSERNUM4
-**    @RGUSERNUM5
+**    STATE: PROCESSDATA
+**     PASS:  @RGUSERCHR1 Target account and share ID
+**            @RGUSERCHR2 Transfer to/from account and share ID
+**            @RGUSERCHR3 Transfer Amount
+**  PROCESS:  Validate the user option selection, perform requested
```

Comment:
I think you will want to add this part of the comment back, that was removed as NUM1 is used:
"@RGUSERNUM1 PROCESSDATA State: Member option selected"

Reply:
done